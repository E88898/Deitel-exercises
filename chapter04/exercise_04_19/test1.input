1
33
55
73
79
64
14
48
69
8
0
