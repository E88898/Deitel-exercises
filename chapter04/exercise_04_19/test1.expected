Largest is 79
Second largest 73
